### DISCUSSION TITLE
Export Data in filament V 2.*

### DISCUSSION THREAD

**Discussion by Rahul-codoffer** [filamentphp/filament#12200] [General]
Is there any option for exporting the Data ?

**Comment by micraux** [comment]
???

What is "_...the Data Table_"?

Do you mean exporting data?
[https://filamentphp.com/docs/3.x/actions/prebuilt-actions/export](https://filamentphp.com/docs/3.x/actions/prebuilt-actions/export)

**Comment by Rahul-codoffer** [comment]
Yes ,
but how can i achieve same functionality in filament V 2.* , also before exporting i want to apply some filter and according to that filter i want to export data.

**Reply by micraux** [reply]
Maybe with this plugin?

[https://filamentphp.com/plugins/pxlrbt-excel](https://filamentphp.com/plugins/pxlrbt-excel)

**Comment by Rahul-codoffer** [comment]
Thanks !
